addons/website/static/src/components/editor/editor.js

```diff
@@ -79,7 +79,6 @@ export class WebsiteEditorComponent extends Component {
         }
         this.wysiwygOptions.invalidateSnippetCache = false;
         this.websiteService.unblockPreview();
-        this.websiteService.hideLoader();
```

Comment:
I think this should not be removed because there are still "flows" where we are in edit mode after a loading screen. Normally, it is the case when you switch themes from the right panel :wink: 

However, with your commit, we are not in edit mode anymore after switching themes :open_mouth:  Maybe check with PDE if this behavior is ok first, but if not, this line should be kept :wink:

Reply:
Thanks for noticing! After asking, landing out of edit mode in this case is ok for PDE.
I don't think another flow needs this line (`hideLoader()`) after opening the edit panel (I've checked some other potential suspects), so I'll remove it.